src/azureExtensions.ts

```diff
@@ -86,6 +86,17 @@ export const azureExtensions: IAzExtMetadata[] = [
     }
 ];
 
+export const legacyTypeMap: Partial<Record<AzExtResourceType, string>> = {
```

Comment:
Is the plan to start deleting these as we make the other extensions compatible? If so, then okie dokes, but if not, we're missing Container Apps in here.

Reply:
Probably should go ahead and remove StorageAccounts and update the context values appropriately before we forget.